src/tools/storybook-addon-system-switcher/src/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { useGlobals } from '@storybook/api';
+import { IconButton } from '@storybook/components';
+import euLogo from '@ecl/resources-eu-logo/logo--en.svg';
+import ecLogo from '@ecl/resources-ec-logo/logo--en.svg';
+
+export const Switcher = () => {
+  const euActive = process.env.STORYBOOK_SYSTEM;
```

Comment:
A bit tricky part here is that this will be an initial value only because this environment variable will be available only at build time. So loading the first time the contents of build/dist folders it will serve as a sort of a default.

So it might worth treating it as a default case of a last resort. 

```js
const system = window.location.pathname.split('/').filter(p => p)[1] || process.env.STORYBOOK_SYSTEM;
```

I use `window.location.pathname` as a more reliable information source as it's based on runtime.

Reply:
Noticed storybook is using reach/router under the hood and provides some utils which could be useful in working with the paths https://github.com/storybookjs/storybook/blob/5987db49f4264ad88b04b97fe406c3c496e475b5/lib/router/src/utils.ts